models/articles.model.js

```diff
@@ -1,9 +1,18 @@
 const db = require("../db/connection");
 
 exports.selectArticles = () => {
-  return db.query("SELECT * FROM articles;").then(({ rows }) => {
-    return rows;
-  });
+  return db
+    .query(
+      `SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url, COUNT(comments.comment_id) AS comment_count
+      FROM articles
+      LEFT JOIN comments
+      ON articles.article_id = comments.article_id
+      GROUP BY articles.article_id
+      ORDER BY created_at DESC;`
+    )
```

Comment:
Well done with this query! It can be tricky, nice work getting the LEFT JOIN in there!